tests/kvbm_integration/test_determinism_disagg.py

```
@@ -213,6 +213,21 @@ def start_server(self, timeout: int = 300) -> bool:
         # Give frontend time to start up
         time.sleep(5)
 
+        model = os.environ.get(
+            "KVBM_MODEL_ID", "deepseek-ai/DeepSeek-R1-Distill-Llama-8B"
+        )
+
+        # Try to download the model.
+        print("Attempting model download...")
+        try:
+            subprocess.run(
```

Ruff (0.14.3)

`subprocess` call with `shell=True` identified, security issue

(S602)